lib/tasks/budgets.rake

```diff
@@ -13,4 +13,11 @@ namespace :budgets do
 
   end
 
+  namespace :groups do
```

Comment:
I think we don't need this rake task anymore as the `voting_style` is a new attribute and it would automatically add the default `knapsack` value to existing Budgets, as you pointed out previously 👍